stylesheets/manifest.css

```diff
@@ -407,7 +407,7 @@ img.emoji {
 
 .conversation-header {
   background: #f3f3f3;
-  border-bottom: solid 1px #d9d9d9; }
+  border-bottom: solid 1px #dadada; }
```

Comment:
This was changed by `scss` without actually modifying the `.scss` file. Remove this change from the commit?